src/screens/Wallet.js

```diff
@@ -76,7 +85,55 @@ class Wallet extends React.Component {
     });
   }
 
+  /**
+   * Called when user clicks to unregister the token, then opens the modal
+   */
+  unregisterClicked = () => {
+    $('#unregisterModal').modal('show');
+  }
+
+  /**
+   * When user confirms the unregister of the token, hide the modal and execute it
+   */
+  unregisterConfirmed = () => {
+    $('#unregisterModal').modal('hide');
+    tokens.unregisterToken(this.props.selectedToken);
+  }
+
+  /*
+   * We show the administrative tools tab only for the users that one day had an authority output, even if it was already spent
+   *
+   * @return {boolean} If should show administrative tab
+   */
+  shouldShowAdministrativeTab = () => {
+    const walletData = hathorLib.wallet.getWalletData();
+
+    for (const tx_id in this.props.historyTransactions) {
+      const tx = this.props.historyTransactions[tx_id];
+      for (const output of tx.outputs) {
+        // This output is not mine
+        if (!hathorLib.wallet.isAddressMine(output.decoded.address, walletData)) {
+          continue;
+        }
+
+        // This token is not the one of this screen
+        if (output.token !== this.props.selectedToken) {
+          continue;
+        }
+
+        if (hathorLib.wallet.isMintOutput(output) || hathorLib.wallet.isMeltOutput(output)) {
```

Comment:
`walletLib.wallet.isAuthorityOutput`

Reply:
This is true only right now, in the last version of our full node we had another authority output and in the next one we might have. Do you think it's worth it to change it just to prevent doing this `||` and we need to remember to change this in case we create another authority output?

In my opinion we should keep like this.

Reply:
Sure, let's keep like this.